collect-forms.rst

```diff
@@ -16,112 +20,118 @@ In order to fill out forms with survey participants, you must first load blank f
-   
-  - Some questions will :ref:`auto-advance <autoadvance-widget>` after being answered.
-  - `Required questions <http://xlsform.org/#required>`_ will not allow you to advance unless answered.
+  $ adb push path/to/form.xml /sdcard/odk/forms/form.xml
 
-- To **remove a response**, :gesture:`Long Press` on the :term:`question label`. 
+Loading forms from device storage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  
+You can also download forms to your device via a web browser, and move them to the :file:`odk/forms/` directory, using the device's file manager (:menuselection:`Settings -> Storage & USB -> Explore`).
 
+1. Go to the Settings menu (:guilabel:`⚙`) on your device and find :menuselection:`Storage & USB`
 
-For a (mostly) complete guide to form question appearance, see :doc:`form-widgets`.
+   .. figure:: /img/collect-forms/device-settings-storage.* 
+     :alt: The Settings menu on an Android Device. The option *Storage and USB* is circled in red.
+  
+     The settings menu may look different on your device.
 
-.. _completing-form:
+2. From the internal storage screen, select :menuselection:`Explore` to open the file manager.
+     
+   .. figure:: /img/collect-forms/device-settings-storage-explore.* 
+     :alt: The Internal Storage settings menu on an Android device. The option *Explore* is circled in red.
+  
+     The :guilabel:`Explore` option opens a file manager that you can use to move forms into Collect.
 
-Completing a Form
--------------------
+Loading form media
+----------------------
 
-Once you have reached the end of a form, you will have the opportunity to *Save* and *Exit* the form. At this point, you may also:
+If a form :ref:`includes images or other media <image-options>`, those files have to be loaded to the device along with the form.
 
-.. _name-form-instance:
+Media files should be placed in a folder labelled :file:`{form-name}-media`. 
 
-Name the form
-~~~~~~~~~~~~~~~
+- When using ODK Aggregate, the form upload prompt includes instructions to upload the :file:`-media` folder. The files are downloaded automatically when :ref:`fetching forms from Aggregate <in-app-get-blank-forms>`.
+- When using Google Drive, the :file:`-media` folder should be uploaded to the same location as the form.
+- If :ref:`loading forms directly to your device <loading-forms-directly>`, the :file:`-media` folder needs to be placed in the :file:`sdcard/odk/forms` directory, alongside the form itself.
 
-The last form screen provides a default name for the form (defined by the form designer). You can rename it. This name only applies to that particular instance of a completed form (not to the blank form).
 
-The Form Name identifies the form in lists throughout the app. For this reason, a meaningful name may be important to you. After you've saved the name, the form automatically moves to the :guilabel:`Send Finalized Form` section, from where you can send it.
+.. _editing-saved-forms:
 
-.. _finalize-form:
+Editing *Saved* forms
```

Comment:
Any reason why this is italicized?

Reply:
It really should be:

```
:formstate:`Saved`
```
and section titles are always sentence case. It's just weird here because we have usually capitalized form states